.github/workflows/codecov.yml

```diff
@@ -30,9 +30,10 @@ jobs:
         env:
           RUST_BACKTRACE: 1
         run: |
-          cargo tarpaulin  --release  --features fast-runtime \
-           --no-fail-fast --workspace --timeout 3600 --engine llvm \
-           --exclude-files **/mock.rs **/weights.rs **/weights/* --out xml --out html
+          # cargo tarpaulin  --release  --features fast-runtime --fail-under 10 \
+          #  --no-fail-fast --workspace --timeout 3600 --engine llvm \
+          #  --exclude-files **/mock.rs **/weights.rs **/weights/* --out xml --out html
+          cargo tarpaulin --workspace -v --exclude-files **/mock.rs **/weights.rs **/weights/* --out xml
```

Comment:
I think we still need `fast-runtime`. Otherwise would be some integration-tests that will take eons.